backend/kaos_backend/util/validators.py

```diff
@@ -77,11 +90,14 @@ def validate_bundle_structure(cls, directory, req_files):
                     cls.validate_dockerfile(files)
                     cls.validate_model_directory(dirs)
                     model_dir = os.path.join(bundle_root, cls.MODEL)
+                    train_file = os.path.join(model_dir, cls.TRAIN)
```

Comment:
please add logic for `./train` vs. `./serve`

Reply:
No extended for serve